packages/api/src/graphql/peer/peer.private-mutation.ts

```diff
@@ -40,7 +40,7 @@ export const updatePeer = (
   authorise(CanCreatePeer, roles)
 
   const nonEmptyInputs = Object.fromEntries(
-    Object.entries(input).filter(([, value]) => !value || value === false)
+    Object.entries(input).filter(([, value]) => value || value === false)
```

Comment:
Is this intentional?

Reply:
Yeah, the old logic was wrong (was a refactor from master to begin with, so this is not a bug on master)